docs/sources/reference/components/otelcol/otelcol.receiver.syslog.md

```diff
@@ -0,0 +1,276 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.receiver.syslog/
+description: Learn about otelcol.receiver.syslog
+title: otelcol.receiver.syslog
+---
+
+# otelcol.receiver.syslog
+
+{{< docs/shared lookup="stability/public_preview.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+`otelcol.receiver.syslog` accepts syslog messages over the network and forwards them as logs to other `otelcol.*` components.
+It supports syslog protocols [RFC5424][] and [RFC3164][] and can receive data over `TCP` or `UDP`.
+
+{{< admonition type="note" >}}
+`otelcol.receiver.syslog` is a wrapper over the upstream OpenTelemetry Collector `syslog` receiver.
+Bug reports or feature requests will be redirected to the upstream repository, if necessary.
+{{< /admonition >}}
+
+You can specify multiple `otelcol.receiver.syslog` components by giving them different labels.
+
+[RFC5424]: https://www.rfc-editor.org/rfc/rfc5424
+[RFC3164]: https://www.rfc-editor.org/rfc/rfc3164
+
+## Usage
+
+```alloy
+otelcol.receiver.syslog "LABEL" {
+  tcp { ... }
+  udp { ... }
+
+  output {
+    logs    = [...]
+  }
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+| Name                              | Type     | Description                                                        | Default   | Required |
+|-----------------------------------|----------|--------------------------------------------------------------------|-----------|----------|
+| `protocol`                        | `string` | The syslog protocol that the syslog server supports.               | `rfc5424` | no       |
+| `location`                        | `string` | The geographic time zone to use when parsing an RFC3164 timestamp. | `UTC`     | no       |
+| `enable_octet_counting`           | `bool`   | Whether to enable RFC6587 octet counting.                          | `false`   | no       |
+| `max_octets`                      | `int`    | The maximum octets for messages when octet counting is enabled.    | `8192`    | no       |
+| `allow_skip_pri_header`           | `bool`   | Allow parsing records without a priority header.                   | `false`   | no       |
+| `non_transparent_framing_trailer` | `string` | The framing trailer when using RFC6587 Non-Transparent-Framing.    | `nil`     | no       |
+
+The `protocol` argument specifies the syslog format supported by the receiver.
+`protocol` must be one of `rfc5424`, `rfc3164`
+
+The `location` argument specifies a Time Zone identifier. The available locations depend on the local IANA Time Zone database.
+See [this wikipedia entry][tz-wiki] for a non-comprehensive list.
```

Comment:
```suggestion
Refer to the [list of tz database time zones][tz-wiki] in Wikipedia for a non-comprehensive list.
```

We should avoid using `See` for an xref link, and where possible (and where it makes sense) use the article title as the link text.